src/server/Cypher.js

```diff
@@ -62,10 +62,13 @@ router.post("/", async (req, res) => {
     columnNames.forEach((name, i) => {
       dataTypes[name] = columnTypes[i];
     });
-    const currentSchema = await database.getSchema();
-    const isSchemaChanged =
-      JSON.stringify(schema) !== JSON.stringify(currentSchema);
-    schema = currentSchema;
+    let currentSchema = schema;
```

Comment:
Since this var is only used in the `if` block, you should move it into the `if` and keep using `const`.